packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.test.tsx

```diff
@@ -98,6 +98,12 @@ describe('VerticalBarChart snapShot testing', () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  it('Should not render bar values', () => {
+    const component = renderer.create(<VerticalBarChart data={chartPoints} hideValues={true} />);
```

Comment:
Is bar label values enabled by default?

We should add this property in the readme for each chart that we have in examples.

Reply:
Yes, it is enabled by default.

Reply:
added the property in the readme